source/features/tags-dropdown.tsx

```diff
@@ -15,7 +15,7 @@ function changeTabToTags(): void {
 
 function updateLinksToTag(): void {
 	// Change links, which point to the content of each tag, to open the tag page instead
-	for (const anchorElement of select.all<HTMLAnchorElement>('.rgh-tags-dropdown .SelectMenu-list:last-child [href*="/tree/"]')) {
+	for (const anchorElement of select.all<HTMLAnchorElement>('.rgh-tags-dropdown [role="tabpanel"]:last-child [href*="/tree/"]')) {
```

Comment:
The problem was just that `SelectMenu-list` isn't the last child anymore
```suggestion
	for (const anchorElement of select.all<HTMLAnchorElement>('.rgh-tags-dropdown .SelectMenu-list [href*="/tree/"]')) {
```

However we can take this opportunity to make the selector more specific to exclude the branches that also appear in the DOM
```suggestion
	for (const anchorElement of select.all<HTMLAnchorElement>('.rgh-tags-dropdown #tags-menu .SelectMenu-item[href*="/tree/"]')) {
```

Reply:
Sure that also works 👍 
I thought the `:last-child` selector was meant to only select the tags tab. 😄